librz/include/rz_analysis.h

```diff
@@ -651,8 +655,10 @@ typedef struct rz_analysis_var_access_t {
 } RzAnalysisVarAccess;
 
 typedef enum {
+	RZ_ANALYSIS_VAR_STORAGE_STACK,
 	RZ_ANALYSIS_VAR_STORAGE_REG,
-	RZ_ANALYSIS_VAR_STORAGE_STACK
+	RZ_ANALYSIS_VAR_STORAGE_COMPOSITE,
```

Comment:
Good point, opened a new issue for that: https://github.com/rizinorg/rizin/issues/3744

Reply:
Added points about serialization in the PR description under `TODO` section

Reply:
Don't forget this one please

Reply:
I'm not quite sure here. Now I should have implemented serialize and load for `COMPOSITE`. 

Do I still need to complete `EVAL_PENDING`? But that would mean implementing JSON serialize and deserilize for `RzBinDwarfLocation` `RzBinDwarfEvaluation`. I feel a bit silly writing this by hand.

Reply:
@imbillow the problem is that COMPOSITE variable would require changes also in `rz-ghidra`, `jsdec`, `rz-retdec`, also `cutter`. After discussion it seems to be easier done in a separate PR since it spans multiple different areas, see https://github.com/rizinorg/rizin/issues/3744. As for `RzBinDwarfLocation` and `RzBinDwarfEvaluation` serialization I don't think we have another choice right now. cc @thestr4ng3r